Dockerfile

```diff
@@ -1,15 +1,66 @@
+FROM ubuntu:25.04 AS selenium-manager-image
+RUN apt-get update \
+  && apt-get install -y --no-install-recommends \
+    ca-certificates curl jq wget \
+  && rm -rf /var/lib/apt/lists/*
+
+ARG SELENIUM_MANAGER_VERSION=latest
+RUN SELENIUM_MANAGER_VERSION_FULL=$(curl -s https://api.github.com/repos/SeleniumHQ/selenium_manager_artifacts/releases/${SELENIUM_MANAGER_VERSION} | jq -r .name) \
+  && wget --no-verbose https://github.com/SeleniumHQ/selenium_manager_artifacts/releases/download/${SELENIUM_MANAGER_VERSION_FULL}/selenium-manager-linux \
+  && chmod +x selenium-manager-linux \
+  && mv selenium-manager-linux /usr/local/bin/selenium-manager
+
+ARG CHROME_VERSION="latest"
+ARG FIREFOX_VERSION="latest"
+# Note: geckodriver version needs to be updated manually
+ARG GECKODRIVER_VERSION="0.34.0"
+
+RUN if [ $FIREFOX_VERSION = "latest" ]; then SE_FIREFOX_VERSION="stable"; \
+  elif [ $FIREFOX_VERSION = "nightly-latest" ]; then SE_FIREFOX_VERSION="nightly"; \
+  elif [ $FIREFOX_VERSION = "devedition-latest" ]; then SE_FIREFOX_VERSION="dev"; \
+  elif [ $FIREFOX_VERSION = "esr-latest" ]; then SE_FIREFOX_VERSION="esr"; \
+  else SE_FIREFOX_VERSION=${FIREFOX_VERSION}; \
+  fi \
+  && export SE_FIREFOX_VERSION \
+  && export SE_GECKODRIVER_VERSION=${GECKODRIVER_VERSION} \
```

Comment:
According to the Selenium Manager [doc](https://www.selenium.dev/documentation/selenium_manager/), it can infer the correct driver version when the browser version is specified. So I think we can skip setting the driver version?

> Driver version discovery. With the discovered browser version, the proper driver version is resolved.

Reply:
Applied it! - cf1674726f8f7cbbb36a90a51439df9488394aa5